scripts/Psake/Saritasa.GitTasks.ps1

```diff
@@ -67,7 +67,7 @@ Task delete-merged-branches -description 'Delete merged remote-tracking branches
     }
 
     $condition = { $_.Merged -eq $true -and $_.LastCommitDate -lt $startDate }
-
+    git fetch --prune
```

Comment:
Exec { git fetch --prune }

Reply:
FIx version and exec.

About `git branch -a --merged`:
Tag `-a` show all branches and remove develop, though  where `{ $_ -ne '* develop' }` : 
```
Executing delete-merged-branches
Deleted branch develop (was 40d6d89).
Deleted branch feature/01 (was 40d6d89).
Deleted branch feature/02 (was 40d6d89).
Deleted branch feature/03 (was 1d13210).
error: branch '* feature/04' not found.
Error: 22.08.2018 16:21:47:
 +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~ [<<==>>] Exception: Exec: Error executing command  git.exe branch -d $branch .
```

Reply:
look's like `Get-GitFlowStatus -BranchType Feature` worked more stable, then `git branch -a --merged origin/develop | where { $_ -ne '* develop' } | ForEach-Object { $_.Trim() }`.
This script works strange because it hides the local develop, if at the moment checkout -> develop. If you run the command at checkout -> feature, then delete the local develop.
And the tag `-a` displays still remote branches, and they should be separately orbased.

```
PS test-psake-script.1> git branch --merged origin/develop | where { $_ -ne '* develop' } | ForEach-Object { $_.Trim() }
feature/01
feature/02
feature/03
feature/04
PS test-psake-script.1> git branch -a --merged origin/develop | where { $_ -ne '* develop' } | ForEach-Object { $_.Trim() }
feature/01
feature/02
feature/03
feature/04
remotes/origin/HEAD -> origin/develop
remotes/origin/develop
remotes/origin/feature/01
remotes/origin/feature/02
remotes/origin/feature/03
remotes/origin/feature/04
```